libfoundation/src/foundation-string.cpp

```diff
@@ -501,6 +501,98 @@ bool MCStringCreateMutable(uindex_t p_initial_capacity, MCStringRef& r_string)
 
 ////////////////////////////////////////////////////////////////////////////////
 
+struct MCStringEncodingName
+{
+    const char *m_name;
+    MCStringEncoding m_encoding;
+};
+
+// These names must match the formatting of character encoding names as output
+// by the algorithm described in section 1.4 of the Unicode Consortium TR22.
+static MCStringEncodingName MCStringEncodingNames[] =
```

Comment:
I think this covered by the standard name processing that occurs - Fraser can confirm :)

Sent from my iPhone

> On 16 Jan 2015, at 08:56, Peter TB Brett <notifications@github.com> wrote:
> 
> In libfoundation/src/foundation-string.cpp:
> 
> > @@ -501,6 +501,98 @@ bool MCStringCreateMutable(uindex_t p_initial_capacity, MCStringRef& r_string)
> >  
> >  ////////////////////////////////////////////////////////////////////////////////
> >  
> > +struct MCStringEncodingName
> > +{
> > +    const char *m_name;
> > +    MCStringEncoding m_encoding;
> > +};
> > +
> > +// These names must match the formatting of character encoding names as output
> > +// by the algorithm described in section 1.4 of the Unicode Consortium TR22.
> > +static MCStringEncodingName MCStringEncodingNames[] =
> Do you think it would be a good idea to add the "official" hyphenated versions of the encoding names as synonyms, e.g.:
> 
> iso-8859-1
> utf-8
> etc.?
> 
> —
> Reply to this email directly or view it on GitHub.
>